frontend/src/scenes/web-analytics/webAnalyticsLogic.tsx

```diff
@@ -882,6 +882,7 @@ export const webAnalyticsLogic = kea<webAnalyticsLogicType>([
                                                   kind: NodeKind.WebExternalClicksTableQuery,
                                                   properties: webAnalyticsFilters,
                                                   dateRange,
+                                                  compareFilter, // NOTE: Backend is not using this yet
```

Comment:
@robbie-c Thoughts on whether this is actually helpful? Does it make sense to compare outbound link clicks? The backend doesn't compare it at all, it's always just current data

It's not hard to implement, but I'm trying to get a feeling here on how this could be helpful. I think it is, but I wanna hear from you.

Reply:
Yeah I think it is, but fine if we don't include it in this PR

Reply:
I've included it :)